templates/taxi/car_detail.html

```diff
@@ -1,12 +1,17 @@
 {% extends "base.html" %}
 
 {% block content %}
+  <h1>
+    Car: {{ car.model }}
+  </h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  <h2>Drivers</h2>
   <hr>
   <ul>
   {% for driver in car.drivers.all %}
```

Comment:
The method `car.drivers.all` should be called with parentheses: `car.drivers.all()`. This is necessary to execute the method and retrieve the list of drivers.